lib/usd/utils/MergePrims.cpp

```diff
@@ -789,6 +790,63 @@ bool mergeDiffPrims(
     return SdfCopySpec(srcLayer, srcPath, dstLayer, dstPath, copyValue, copyChildren);
 }
 
+//----------------------------------------------------------------------------------------------------------------------
+/// Create any missing parents as "over". Parents may be missing because we are targeting a
+/// different layer than where the destination prim is authored. The SdfCopySpec function does not
+/// automatically create the missing parent, unlike other functions like UsdStage::CreatePrim.
+///
+/// Unfortunately, USD won't actually author "over" (or even "def") until we set at least one
```

Comment:
That's not true.  See `SdfJustCreatePrimInLayer()`
https://graphics.pixar.com/usd/release/api/prim_spec_8h.html#af1812eef21037f6226a8b66765831542
This is what I think you should be using.

Reply:
That function would create a valid prim, and only its parents would be "over". We just need to create the parent over, The function is *almost* what we need, but not quite :)

Reply:
`SdfJustCreatePrimInLayer` creates the prim as an over, with empty type.  Just use the parent of the path and you should be fine.

Reply:
I'll try it. The description of the function is unclear if the prim is created as an over or only its parents.

Reply:
Juts looked at the implementation code in USD and indeed, it creates even the final prim as an over